tests/Feature/Student/StudentDetailControllerTest.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Tests\Feature;
+
+use Illuminate\Foundation\Testing\DatabaseTransactions;
+use Illuminate\Foundation\Testing\WithFaker;
+use Tests\TestCase;
+use App\Models\Resume;
+use App\Service\StudentDetailsService;
+use App\Models\Student;
+
+class StudentDetailControllerTest extends TestCase
+{
+    use DatabaseTransactions;
+
+    public function test_student_details_found()
+    {
+        $studentDetailsService = $this->createMock(StudentDetailsService::class);
+
+        $student = Student::factory()->create();
+    
+        $studentId = $student->id;
+        
+        $studentDetails = Resume::factory()->create();
+        $studentDetailsService->expects($this->once())
+                              ->method('execute')
+                              ->with($studentId)
+                              ->willReturn($studentDetails);
+
+        $this->app->instance(StudentDetailsService::class, $studentDetailsService);
+
+        $response = $this->get(route('student.detail', ['id' => $studentId]));
+
+        $response->assertStatus(200);
+
+        $response->assertJsonStructure();
+
+        /*$response->assertJsonStructure([
+            'student' => [
+                'fullname',
+                'subtitle',
+                'social_media' => [
+                    'github' => ['url'],
+                    'linkedin' => ['url']
+                ],
+                'about',
+                'tags' => [
+                    '*' => [
+                        'id',
+                        'name'
+                    ]
+                ]
+            ]
+        ]);*/
```

Comment:
If you're not going to use this block of code, it's better to delete it.

Reply:
I forgot to delete it.